plugins/wazuh-core/server/initialization/server-api.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  PLUGIN_APP_NAME,
+  PLUGIN_PLATFORM_WAZUH_DOCUMENTATION_URL_PATH_TROUBLESHOOTING,
+} from '../../common/constants';
+import { webDocumentationLink } from '../../common/services/web_documentation';
+import { version as appVersion } from '../../package.json';
+
+export const initializationTaskCreatorServerAPIConnectionCompatibility = ({
+  taskName,
+}: {
+  taskName: string;
+}) => ({
+  name: taskName,
+  async run(ctx) {
+    try {
+      ctx.logger.debug(
+        'Starting check server API connection and compatibility',
+      );
+      const results = await ServersAPIConnectionCompatibility(ctx);
```

Comment:
You could add this in your tests. You could add a test that checks if ctx.manageHosts.get was called with the correct arguments.

Reply:
I used a different approach, checking the logger instead.